site/docs-md/guides/ionic-framework-app.md

```diff
@@ -0,0 +1,250 @@
+Next, we need to tell Angular to trust the dynamic image data. To do this, inject `DomSanitizer` via the Constructor and use `sanitizer.bypassSecurityTrustResourceUrl()` to allow the image data to be displayed in our app:
+
+```typescript
+import { DomSanitizer, SafeResourceUrl } from '@angular/platform-browser';
+
+export class Tab2Page {
+  photo: SafeResourceUrl;
+
+  constructor(private sanitizer: DomSanitizer) {  }
+}
+```
+
+Here's the complete Camera implementation:
+
+```typescript
+import { Component } from '@angular/core';
+import { Plugins, CameraResultType, CameraSource } from '@capacitor/core';
+import { DomSanitizer, SafeResourceUrl } from '@angular/platform-browser';
+
+@Component({
+  selector: 'app-tab2',
+  templateUrl: 'tab2.page.html',
+  styleUrls: ['tab2.page.scss']
+})
+export class Tab2Page {
+  photo: SafeResourceUrl;
+
+  constructor(private sanitizer: DomSanitizer) {  }
+
+  async takePicture() {
+    const image = await Plugins.Camera.getPhoto({
+      quality: 100,
+      allowEditing: false,
+      resultType: CameraResultType.Base64,
+      source: CameraSource.Camera
+    });
+
+    this.photo = this.sanitizer.bypassSecurityTrustResourceUrl(image && (image.base64Data));
+  }
+}
+```
+
+Let's give this a try in the browser:
+
+```shell
+ionic serve
+```
+
+Open the Developer Tools and select the `Console` tab. Then, navigate over to Tab Two, then click the newly created Camera fab button. An error appears in the Console:
+
+`ERROR Error: Uncaught (in promise): TypeError: cameraModal.present is not a function`
+
+Some Capacitor plugins, including the Camera, provide the web-based functionality and UI via the Ionic [PWA Elements library](https://github.com/ionic-team/ionic-pwa-elements). It's a separate dependency, so install it using the Terminal (cancel the `ionic serve` command currently running first):
+
+```shell
+npm install @ionic/pwa-elements
+```
+
+**Note:** There's currently an issue with importing the Ionic PWA elements library in an Ionic Angular app. In the meantime, include this script tag in `src/index.html`: 
+
+```html
+<head>
+  <!-- Other tags -->
+
+  <script src="https://unpkg.com/@ionic/pwa-elements@1.0.2/dist/ionicpwaelements.js"></script>
+</head>
+```
+
+With that appropriately configured now, re-run `ionic serve` and click the Camera fab button. If your computer has a webcam of any sort, a modal window displays and you can take a photo!
+
+![Camera API on the web](/assets/img/docs/guides/ionic-framework/camera-web.png)
+*"I'm pretty handsome." - The Author*
+
+Next up, let's run this app on iOS and Android.
+
+## iOS
+Capacitor iOS apps are configured and managed through Xcode, with dependencies managed by CocoaPods. Before running this app on an iOS device, there's a couple of steps to complete.
+
+From the Terminal, run the Capacitor `sync` command, which updates all native code dependencies as well as copies all web assets (the Ionic Angular app in this case) into the native iOS project:
```

Comment:
Sync is only needed when you install new plugins, so use copy instead to just copy the app assets (www folder), it's way faster.